network/channels/channel.go

```diff
@@ -17,20 +17,20 @@ func (c Channel) String() string {
 }
 
 // Len returns length of the ChannelList in the number of stored Channels.
-// It satisfies the sort.Interface making the ChannelList sortable.
+// It satisfies the sort.Type making the ChannelList sortable.
```

Comment:
?

Reply:
[Fixed]Goland change String and Type in a bunch of unrelated places when I renamed the fields on the MsgAuthConfig